m4/ax_python_module.m4

```diff
@@ -4,13 +4,15 @@
 #
 # SYNOPSIS
 #
-#   AX_PYTHON_MODULE(modname[, fatal])
+#   AX_PYTHON_MODULE(modname[, fatal, python])
```

Comment:
It would be better to leave this macro as-is, and instead set `$PYTHON` in `configure.ac`, since the macro already provides you that option to override the python executable.

Something like:
```m4
AC_PATH_PROG([PYTHON], [python3 python])
AX_PROG_PYTHON_VERSION([3.0], [], [AC_MSG_ERROR([You must have Python 3.])])
```
Checking for a Python 3 executable named `python` and using [`ax_prog_python_version.m4`](https://github.com/djmitche/autoconf-archive/blob/master/m4/ax_prog_python_version.m4) to make sure it is not Python 2, I'm not sure is necessary though. You might be able to get away with just `AC_PATH_PROG([PYTHON], [python3])`.